src/main/webapp/app/exam/manage/student-exams/student-exam-detail-table-row/student-exam-detail-table-row.component.html

```diff
@@ -31,7 +31,7 @@
     </ng-template>
 </td>
 <td>
-    <div *ngIf="course?.isAtLeastInstructor && exercise.type != ExerciseType.QUIZ">
+    <div *ngIf="course?.isAtLeastInstructor && exercise.type !== ExerciseType.QUIZ && studentExam.submitted">
         <div *ngIf="result && exercise.type != ExerciseType.PROGRAMMING">
```

Comment:
Can we change the `!=` to `!==` here while we're changing this? (Same for line 48.)

Reply:
Of course, done